package.json

```diff
@@ -48,7 +48,7 @@
     "formik": "^0.10.4",
     "hast-util-sanitize": "^1.1.2",
     "keytar": "^4.2.1",
-    "lbry-redux": "lbryio/lbry-redux#7759bc6e8c482bed173d1f10aee6f6f9a439a15a",
+    "lbry-redux": "lbryio/lbry-redux#121ba56f47fff05531e27a7c99d7d417e79cd3ee",
```

Comment:
Not blocking - but what's it look like to package this properly? Does it make dev cycle more work / more annoying? Possibly worth filing.

Reply:
Not really sure. It's pretty simple right now, just build `lbry-redux` push the new build and grab the latest commit. I would think it would be more work to actually package it.

I think that could be valuable in the future, but while we are adding lots of stuff, I think this is working fine.